sdk/core/src/mock_transaction.rs

```diff
@@ -0,0 +1,22 @@
+#[cfg(feature = "mock_transport_framework")]
```

Comment:
Are these necessary? The module itself will not be compiled if the feature isn't on

Reply:
No it's not! Good catch!